services/core-web/src/components/Forms/permits/PreDraftPermitForm.js

```diff
@@ -67,16 +68,11 @@ export const PreDraftPermitForm = (props) => {
             Cancel
           </Button>
         </Popconfirm>
-        <NOWActionWrapper permission={Permission.EDIT_PERMITS}>
-          <Button
-            className="full-mobile"
-            type="primary"
-            htmlType="submit"
-            loading={props.submitting}
-          >
-            Start Draft Permit
-          </Button>
-        </NOWActionWrapper>
+        {/* <NOWActionWrapper permission={Permission.EDIT_PERMITS}> */}
```

Comment:
Can the commented-out code in this file be removed?

Reply:
@mmumby ?